src/clean_air/visualise/dataset_renderer.py

```diff
@@ -129,15 +145,15 @@ def linear_interpolate(self):
             interpolated data.
             """
         point_cube = self.data.extract_point((self.x, self.y))
-
+        # Identify and remove any single-point coordinates:
+        scalar_coords = []
+        for coord in point_cube.dim_coords:
+            if len(coord.points) == 1:
+                scalar_coords.append(coord)
+        for coord in scalar_coords:
+            point_cube = point_cube.collapsed(coord.standard_name, iris.analysis.MEAN)
         return point_cube
 
-    def track(self, crs=None):
```

Comment:
Removing this has broken the flight timeseries functionality/notebook, as this was the public-facing method used. You're right in that this method doesn't do much in itself, but I think good to leave it here as all of the Timeseries functions are used in the same way - it'll be confusing for the user that one of the four types of timeseries plots needs to be used in a different way.
(FYI this method is also outdated, the newest version is on branch 'fix_tests' - we can rebase afterwards though)